dev-tools/jenkins_ci.ps1

```diff
@@ -32,7 +32,7 @@ $env:TEST_COVERAGE = "true"
 $env:RACE_DETECTOR = "true"
 
 # Install mage from vendor.
-exec { go install github.com/elastic/beats/vendor/github.com/magefile/mage } "mage install FAILURE"
+exec { go install -mod=vendor github.com/magefile/mage } "mage install FAILURE"
```

Comment:
Just for my understanding, this change is not really related to the `_template` path change, right? It's just another change needed when moving to go modules?